AppCenterDataStorage/AppCenterDataStorageTests/MSDBDocumentStoreTests.m

```diff
@@ -202,6 +200,37 @@ - (void)testReadUserDocumentFromLocalDatabaseNotFound {
   XCTAssertEqualObjects(documentWrapper.documentId, documentId);
 }
 
+- (void)testUpsertReplacesCorrectlyInAppStorage {
+
+  // If
+  MSDocumentWrapper *expectedDocumentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestDocument"]
+                                                                           documentType:[MSDictionaryDocument class]];
+  MSWriteOptions *writeOptions = [[MSWriteOptions alloc] initWithDeviceTimeToLive:MSDataStoreTimeToLiveInfinite];
+
+  // When
+  // Upsert twice to ensure that replacement is correct.
```

Comment:
This is no longer upserting twice in this part, comment should be corrected

Reply:
Just meant in general we do it twice, not necessarily in that block i guess